### PR TITLE
add provisioning doc with tags and update navigation

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by pwright** [comment]
@odra  can you take a look?

File: roles/provision-buildfarm-apb/templates/jenkins-persistent.json.j2 [hunk 1]
```diff
@@ -90,15 +90,15 @@
                 "name": "jenkins",
                 "image": "${JENKINS_IMAGE_NAME}:${JENKINS_IMAGE_VERSION}",
                 "readinessProbe": {
-                  "timeoutSeconds": 3,
+                  "timeoutSeconds": 10,
```
Comment (odra) [inline roles/provision-buildfarm-apb/templates/jenkins-persistent.json.j2]: I think the branch needs a rebase - this change was merged today

**Comment by pwright** [comment]
@odra  rebased

**Comment by pwright** [comment]
@odra  anything else I need to do here?

**Review by odra — APPROVED** [review]

**Pull request merged**